content/docs/04-clusters/03-edge/04-edgeforge-workflow/03.5-palette-canvos.md

```diff
@@ -0,0 +1,658 @@
+
+<Tabs>
+
+<Tabs.TabPane tab="Basic" key="basic_create_artifacts">
+
+
+# Prerequisites
+To complete this basic guide, you will need the following items:
+<br/>
+
+* A physical or virtual Linux machine with **AMD64** (also known as **x86_64**) processor architecture. You can issue the following command in the terminal to check your processor architecture. 
+  <br/>
+
+  ```bash
+  uname -m
+  ```
+
+* Minimum hardware configuration of the Linux machine:
+  - 4 CPU
+  - 8 GB memory
+  - 50 GB storage
+ 
+
+* [Git](https://cli.github.com/manual/installation). You can ensure git installation by issuing the `git --version` command.
+
+
+* [Docker Engine](https://docs.docker.com/engine/install/) version 18.09.x or later. You can use the `docker --version` command to view the existing Docker version. You should have root-level or `sudo` privileges on your Linux machine to create privileged containers.   
+<br />
+
+  <WarningBox>
+
+  Issue the following command to check if you can create privileged containers on your Linux machine.
+  <br />
+
+  ```bash
+  docker run --privileged hello-world
+  ```
+  If the command above fails, create a `docker` user group and add the current user to it using the instructions available [here](https://docs.docker.com/engine/install/linux-postinstall/). Alternatively, if you do not want to create a user group, ensure you can issue Docker commands with `sudo` privileges on your Linux machine.
+
+  </WarningBox>
+
+* A [Spectro Cloud](https://console.spectrocloud.com) account. If you have not signed up, you can sign up for a [free trial](https://www.spectrocloud.com/free-tier/).
+
+
+* Palette registration token for pairing Edge hosts with Palette. You will need tenant admin access to Palette to generate a new registration token. For detailed instructions, refer to the [Create Registration Token](/clusters/edge/site-deployment/site-installation/create-registration-token) guide.
+
+
+# Instructions
+
+Use the following instructions on your Linux machine to create all the required Edge artifacts quickly. 
+<br />
+
+1. Check out the [CanvOS](https://github.com/spectrocloud/CanvOS.git) GitHub repository containing the starter code. 
+  <br />
+
+  ```bash
+  git clone https://github.com/spectrocloud/CanvOS.git
+  ```
+
+
+2. Change to the **CanvOS/** directory. 
+  <br />
+
+  ```bash
+  cd CanvOS
+  ```
+
+
+3. View the available [git tags](https://github.com/spectrocloud/CanvOS/tags).
+<br />
+
+  ```bash
+  git tags
+  ```
+
+
+4. Check out the repository tag that matches your Palette product version. Your Palette product version is displayed in the top left corner, as shown in the screenshot below. 
+
+  ![Screenshot displaying the Palette product version.](/tutorials/palette-canvos/clusters_edge_palette-canvos_palette-version.png)
+
+  It is essential to check out a matching git tag so the generated Edge installer will be compatible with your Palette product version. This guide uses **v3.4.3** tag as an example. 
```

Comment:
The first sentence is not written in an active voice. Address the user, speak to them directly. Starting a sentence with "is is essential" automatically puts a distance between the user and us.  If you're giving a recommendation, then say "We recommend"... otherwise it feels like we are trying to put space between what we say and what the user does.